Sources/TokamakCore/Views/NavigationView.swift

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+public struct NavigationView<Content>: View where Content: View {
+  let content: Content
+
+  @State var destination = AnyView(VStack {
+    HStack { Spacer() }
+    Spacer()
+    Text("No Selection")
```

Comment:
**suggestion:** as this text is not how SwiftUI does it, maybe it should be moved to the demo code instead of residing in `TokamakCore`?

Reply:
Should I just go with `AnyView(EmptyView())` instead?

Reply:
Sure, I hope there are no visible differences from the SwiftUI behaviour then

Reply:
The demo crashes on iOS 13 now but on iOS 14 simulator there’s an EmptyView.